WolvenKit.RED4/Archive/IO/CR2WReader.File.cs

```diff
@@ -73,13 +73,13 @@ public EFileReadErrorCodes ReadFileInfo(out CR2WFileInfo? info, ILoggerService?
     public EFileReadErrorCodes ReadFile(out CR2WFile? file, bool parseBuffer = true)
     {
         var result = ReadFileInfo(out var info);
-        if (result == EFileReadErrorCodes.NoCr2w)
+        if (result == EFileReadErrorCodes.NoCr2w || info is null || info.StringDict.Count == 0)
```

Comment:
Just as info: `info` can't be `null` below, so the `!` should be enough. But the null check to suppress the compiler warning below is also fine.

But adding `info.StringDict.Count == 0` here is technicly wrong. If the previous checks are false, it is a CR2W file and the empty StringDict has another reason. The return value of `NoCr2w` is wrong then.